label_studio/core/utils/common.py

```diff
@@ -750,3 +750,17 @@ def empty(*args, **kwargs):
 def get_ttl_hash(seconds: int = 60) -> int:
     """Return the same value within `seconds` time period"""
     return round(time.time() / seconds)
+
+def is_enterprise():
+    """Determine if the current Label Studio instance is Enterprise (LSE) or open-source (LSO).
+
+    Returns
+    -------
+    bool
+        True if running Label Studio Enterprise, False if running open-source Label Studio
+    """
+    try:
+        import label_studio_enterprise
+        return True
+    except ImportError:
+        return False
```

Comment:
Proposing that we invert this, because elsewhere in the code we check for enterprise meaning "enterprise, as opposed to starter cloud" rather than "enterprise, as opposed to LSO"

```suggestion
def is_community():
    """Determine if the current Label Studio instance is the community edition (aka LSO).

    Returns
    -------
    bool
        True if running open-source Label Studio, False otherwise.
    """
    try:
        import label_studio_enterprise
        return False
    except ImportError:
        return True
```

Reply:
GH wouldn't let me apply this directly, but I made these changes. Good suggestion, I totally agree with the thinking here 👍